java/src/org/openqa/selenium/json/Json.java

```diff
@@ -27,20 +27,117 @@
+ *   <li>Facilities to deserialize custom data types:
+ *     <ul>
+ *       <li>Classes that declare setter methods that adhere to the {@code JavaBean} specification.</li>
+ *       <li>Classes that declare a {@code fromJson(JsonInput)} method.</li>
+ *     </ul>
+ *   </li>
+ * </ul>
+ *
+ * The standard types supported by built-in processing:
+ * <ul>
+ *   <li><b>Numeric Types</b>:
+ *     <ul>
+ *       <li>{@link java.lang.Byte Byte}</li>
+ *       <li>{@link java.lang.Double Double}</li>
+ *       <li>{@link java.lang.Float Float}</li>
+ *       <li>{@link java.lang.Integer Integer}</li>
+ *       <li>{@link java.lang.Long Long}</li>
+ *       <li>{@link java.lang.Short Short}</li>
+ *     </ul>
+ *   </li>
+ *   <li><b>Collection Types</b>:
+ *     <ul>
+ *       <li>{@link java.util.List List}</li>
+ *       <li>{@link java.util.Set Set}</li>
+ *     </ul>
+ *   </li>
+ *   <li><b>Standard Java Types</b>:
+ *     <ul>
+ *       <li>{@link java.util.Map Map}</li>
+ *       <li>{@link java.lang.Boolean Boolean}</li>
+ *       <li>{@link java.lang.String String}</li>
+ *       <li>{@link java.lang.Enum Enum}</li>
+ *       <li>{@link java.net.URI URI}</li>
+ *       <li>{@link java.net.URL URL}</li>
+ *       <li>{@link java.util.UUID UUID}</li>
+ *       <li>{@link java.time.Instant Instant}</li>
+ *       <li>{@link java.lang.Object Object}</li>
+ *     </ul>
+ *   </li>
+ * </ul>
+ *
+ * The <b>Json</b> API also handles Selenium {@link org.openqa.selenium.Capabilities Capabilities} objects:
+ * <pre>
+ *   String json = "{\"browserName\":\"htmlunit\",\"version\":\"chrome\"}";
+ *   Capabilities capabilities = new Json().toType(json, Capabilities.class);
+ * </pre>
+ *<p>
+ * You can serialize objects for which no explicit coercer has been specified, and the <b>Json</b> API will use a
+ * generic process to provide best-effort JSON output. For the most predictable results, though, it's best to
+ * provide a {@code toJson()} method for the <b>Json</b> API to use for serialization. This is especially beneficial
+ * for objects that contain transient properties that should be omitted from the JSON output.
+ * <p>
+ * You can deserialize objects for which no explicit coercer has been defined via the <b>ObjectCoercer</b>. Note
+ * that the data type of the result will be {@code Map<String,?>}, which means that you'll need to perform type
+ * checking and casting every time you extract an entry value from the result. For this reason, it's best to declare
+ * a specific {@code fromJson()} method in every type you need to deserialize.
+ * <p>
+ *
+ * @see JsonTypeCoercer
+ * @see JsonInput
+ * @see JsonOutput
+ */
 public class Json {
+  /** This constant declares the value of the {@code Content-Type}
+   *  headers of HTTP requests and responses with JSON entities */
   public static final String JSON_UTF_8 = "application/json; charset=utf-8";
 
+  /** This constant specifies deserializing as {@code List<Map<String, Object>} */
```

Comment:
Same applies here and elsewhere: we don't need to tell people a constant is a constant

Reply:
Done!